.plzconfig.localremote

```diff
@@ -6,3 +6,4 @@ NumExecutors = 10
 TokenFile = grpcutil/token.txt
 Secure = false
 DisplayUrl = http://localhost:7779
+Instance = mettle
```

Comment:
This is a minor one I found while noodling about; without it the client doesn't generate very nice URLs. It doesn't matter much otherwise since we don't do a lot with the instance name.